src/VirtoCommerce.CatalogModule.Web/Scripts/blades/editorialReview-detail.js

```diff
@@ -28,8 +28,9 @@ angular.module('virtoCommerce.catalogModule')
             var existReview = _.find(blade.item.reviews, function (x) { return x == blade.origEntity; });
             if (!existReview) {
                 blade.item.reviews.push(blade.origEntity);
-            };
+            }
             angular.copy(blade.currentEntity, blade.origEntity);
+            blade.currentEntity.isInherited = false;
```

Comment:
Please remove code duplication by calling the `saveChanges()` function, which is declared later in this file.